data/schema.yaml

```
@@ -203,9 +203,9 @@ components:
           properties:
             headlines:
               type: object
-              # required:
-              #   - seoHeadline   currently not in the test optimo datafeed
-              #   - promoHeadline   currently not in the test optimo datafeed
+              required:
+                - seoHeadline
+                - promoHeadline
```

Is promo headline being sent by Optimo, if not uncommenting this line would mean our validator has a hard requirement on data that is in our local fixtures and therefore passes but isn't being produced in the actual Ares feed

---

```suggestion
              #   - promoHeadline   currently not in the test optimo datafeed
```

Confirmed with Cameron and Iain that Optimo is not yet producing the `promoHeadline`

---

Making changes myself as @jtart is out sick